dotnet/samples/Connectors/EventDrivenTcpThermostatConnector/KubernetesResources/connector-template.yaml

```diff
@@ -14,7 +14,8 @@ spec:
       managedConfigurationType: imageConfiguration
       imageConfigurationSettings:
         image: eventdriventcpthermostatconnector:latest
-        imagePullPolicy: Never
+        imagePullPolicy: Never # This sample image is built locally, so no need to pull
+        replicas: 1
```

Comment:
It took me a bit of digging to find where this was configurable, so I figure including it in our samples will spare our users from the same